### ISSUE TITLE
Estimate time of fit/simulate

### ISSUE THREAD

**Issue by dmey** [vinecopulib/vinecopulib#524]
When working with large data, the fitting or simulation process may take several hours/days to complete. Are there any plans to add some time estimates? I do not know how complex it is to estimate the time required based on configurations etc (I guess quite), but perhaps just adding some logging info based on the iteration (e.g. iteration_number/total number of iterations) may be enough.

**Comment by tvatter** [comment]
We haven't really thought of that. We could achieve this by improving what is displayed when using `show_trace`. We could either add some additional logging info as the fit/simulation progresses or some kind of progress bar. Any thoughts?

**Comment by dmey** [comment]
I don't have a particular preference about this and I think simply showing the progress bar would be good enough, at least to start with. One thing though, given that you provide binaries for Python and R, it would be great if we can keep this as a state or return a counter that can be used by other libraries as well rather than just printing the status bar in C.

As a separate note, I think that when I tried using `show_trace` for the fitting the process took considerably longer -- Could this happen -- I may be wrong about this but.

**Comment by tnagler** [comment]
Yeah printing the trace to the terminal can be very slow for large models. We should think about an optionfor logging to a file and more compressed output for large models.
